### DISCUSSION TITLE
Display Aspect Ratio (DAR) support

### DISCUSSION THREAD

**Discussion by wdsongama** [mifi/lossless-cut#1868] [Ideas]
I noticed that, when I open a DVD file that has 720x576 resolution but is flagged as 16:9, LosslessCut ignores the DAR. I think it's a nice idea to keep the DAR, so I don't need to convert the video file to 1024x576 square pixel before dropping it into the software.

**Comment by mifi** [comment]
by "ignore the DAR" do you mean that the preview is incorrect, or do you mean when you export, the exported file gets the wrong DAR?

**Reply by wdsongama** [reply]
Both actually. The preview and the exported file only consider the resolution, but not the DAR, so the image looks "squeezed". I only exported the frames as images though.
